src/WebJobs.Script/Description/Rpc/WorkerLanguageInvoker.cs

```diff
@@ -62,7 +63,7 @@ protected override async Task<object> InvokeCore(object[] parameters, FunctionIn
             ScriptInvocationContext invocationContext = new ScriptInvocationContext()
             {
                 FunctionMetadata = Metadata,
-                BindingData = context.Binder.BindingData,
+                BindingData = context.Binder.BindingData.Where(kvp => !(kvp.Value is DefaultHttpRequest)).ToDictionary(a => a.Key, b => b.Value),
```

Comment:
This is also a breaking change!

Reply:
@mhoeger The "inputs" member of the invocationContext still has the DefaultHttpRequest from the binding (I'm assuming this is the reason all of the manual and automatic tests have been passing). I'm not sure if it's duplicates are still required in the BindingData, unless I'm missing something else?

Reply:
Reverted this. Taking this as a separate PR / capability as discussed.